reana_server/fetcher.py

```diff
@@ -409,12 +421,12 @@ def get_fetcher(
                 "The provided specification doesn't have a valid file extension"
             )
 
-    if parsed_url.extension == ".git":
+    if parsed_url.netloc == "github.com":
+        return _get_github_fetcher(parsed_url, output_dir, spec)
+    elif parsed_url.extension == ".git":
```

Comment:
Just a quick question: is there still a use case for having this check? If we take an example of this url (`https://github.com/reanahub/reana-demo-root6-roofit.git`), in this case `_get_github_fetcher()` would be called. So is there a case in which `WorkflowFetcherGit()` would be returned here while `_get_github_fetcher()` would not being called?

Reply:
I guess other repositories, for example, from GitLab.

Reply:
This check is still useful when dealing with Git repositories that are not on GitHub (e.g GitLab). However, in this case it is currently not possible to specify a particular commit/tag/branch to use, so perhaps we should improve this in the future.

Reply:
Thanks, makes sense